hack/images.sh

```
@@ -1,2 +1,2 @@
-MKPJ_IMG=us-docker.pkg.dev/k8s-infra-prow/images/mkpj:v20260429-8f72e9c5a
-MKPOD_IMG=us-docker.pkg.dev/k8s-infra-prow/images/mkpod:v20260429-8f72e9c5a
+MKPJ_IMG=us-docker.pkg.dev/k8s-infra-prow/images/mkpj:v20260504-92d6574a4
```

Shellcheck (0.11.0)

[warning] MKPJ_IMG appears unused. Verify use (or export if used externally).

(SC2034)